### PR TITLE
Add Auth0 MFA Risk Assessment Enabled

### PR DESCRIPTION
**Pull request by papanikge**
### Background

Detection for Auth0 MFA Policies
Fires an `HIGH` severity alert

### Changes

* Adds `rules/auth0_rules/auth0_mfa_risk_assessment_enabled.py` & `rules/auth0_rules/auth0_mfa_risk_assessment_enabled.yaml` files and udpates the pack.

### Testing

Test with
```
pipenv run panther_analysis_tool test --filter RuleID=Auth0.MFA.Risk.Assessment.Enabled
```

and got:
```
Auth0.MFA.Risk.Assessment.Enabled
	[PASS] Other Event
		[PASS] [rule] false
	[PASS] Risk Assessment Enabled
		[PASS] [rule] true
		[PASS] [title] Auth0 User [user.name@yourcompany.io] enabled mfa risk assessment settings for your organization’s tenant [<NO_P_SOURCE_LABEL_FOUND>].
		[PASS] [dedup] Auth0 User [user.name@yourcompany.io] enabled mfa risk assessment settings for your organization’s tenant [<NO_P_SOURCE_LABEL_FOUND>].
		[PASS] [alertContext] {"actor": {"email": "user.name@yourcompany.io", "name": "User Name", "user_id": "google-oauth2|105261262156475850461"}, "action": "Updates risk assessment configs"}

--------------------------
Panther CLI Test Summary
	Path: .
	Passed: 1
	Failed: 0
	Invalid: 0
```

### CODE REVIEW

File: rules/auth0_rules/auth0_mfa_risk_assessment_enabled.yml [hunk 1]
```diff
@@ -0,0 +1,355 @@
+AnalysisType: rule
+Description: An Auth0 User enabled the mfa risk assessment setting for your organization's tenant.
+DisplayName: "Auth0 MFA Risk Assessment Enabled"
+Enabled: true
+Filename: auth0_mfa_risk_assessment_enabled.py
+Runbook: Assess if this was done by the user for a valid business reason. Be vigilant when enabling this setting as it's in the best security interest for your organization's security posture.
+Severity: High
```
Comment (calkim-panther) [inline rules/auth0_rules/auth0_mfa_risk_assessment_enabled.yml:7]:
```suggestion
Severity: Info
```
I'd classify this as an Info level alert

**Comment by andrea-youwakim** [comment]
> @andrea-youwakim Wanted to double check if this detection was intended to alert on mfa policy **enabled** or **updated**. Looks like the scoping and this rule are alerting on updates to the config and not on enable.

i think this pr is just a bit poorly worded: https://www.notion.so/pantherlabs/Hackathon-auth0-mfa-risk-assessment-enabled-6410d1ae40fc4b358d998a4b8351a28b?pvs=4 here is the associated documentation for the alert

File: rules/auth0_rules/auth0_mfa_risk_assessment_enabled.yml (hunk 1 above)
Comment (andrea-youwakim) [reply rules/auth0_rules/auth0_mfa_risk_assessment_enabled.yml:7]: agreed, this is in the notion documentation for this detection as well

Comment (andrea-youwakim) [reply rules/auth0_rules/auth0_mfa_risk_assessment_enabled.yml]: committing change

**Review by andrea-youwakim — CHANGES_REQUESTED** [review]
could you actually run the tests for this specific rule id and include them in the pr description? this will help easily confirm desired functionality of the detection content for anyone who views the pr

File: rules/auth0_rules/auth0_mfa_risk_assessment_enabled.yml (hunk 1 above)
Comment (papanikge) [reply rules/auth0_rules/auth0_mfa_risk_assessment_enabled.yml]: Nice, thank you.

**Comment by papanikge** [comment]
@andrea-youwakim Done.

**Review by andrea-youwakim — APPROVED** [review]

**Pull request merged**